alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestAcknowledgeAlertAction.kt

```diff
@@ -100,7 +104,6 @@ class RestAcknowledgeAlertAction(settings: Settings, controller: RestController)
                     .filter(QueryBuilders.termsQuery("_id", alertIds))
             val searchRequest = SearchRequest()
                     .indices(AlertIndices.ALERT_INDEX)
-                    .types(Alert.ALERT_TYPE)
                     .routing(monitorId)
                     .source(SearchSourceBuilder().query(queryBuilder).version(true))
```

Comment:
I believe you need to set [seq_no_primary_term](https://www.elastic.co/guide/en/elasticsearch/reference/7.0/search-request-seq-no-primary-term.html)  to `true` in order to use it inside `onSearchResponse`

Could you confirm it is working as expected ?

Reply:
This does work as expected and is unable to be changed. Since we are moving a created document to a new document the sequence number and primary term is already set for the created document. The new document creation will fail due to version conflict (expecting `primaryTerm` and `seqNo` to be 0 but is actually 1).